package/MDAnalysis/core/universe.py

```diff
@@ -387,11 +387,11 @@ def load_new(self, filename, format=None, in_memory=False, **kwargs):
                                  trj_n_atoms=self.trajectory.n_atoms))
 
         if in_memory:
-            self.transfer_to_memory(kwargs.get("in_memory_frame_interval", 1))
+            self.transfer_to_memory(step=kwargs.get("in_memory_frame_interval", 1))
```

Comment:
variables and options should be named consistently when we make changes like this. Ideally I would like them here also to be called start stop step without any prefix. But that might be confusing loading a normal reader.